site/_js/main.js

```diff
@@ -27,6 +27,7 @@ import './web-components/share-button';
 import './web-components/web-tabs';
 import './web-components/language-select';
 import './third-party/announcement-banner/announcement-banner';
+import './web-components/enhanced-select';
```

Comment:
This is not used on every page, so it is adding bloat to the main js unnecessarily.